pyartcd/pyartcd/pipelines/prepare_release_konflux.py

```
@@ -1666,6 +1667,48 @@ def update_release_jira(self, issue: Issue, template_issue: Issue, template_vars
+
+        # Check 2: bug.yml target_release must include .z
+        try:
+            bug_config = yaml.load(await self.build_data_repo.read_file("bug.yml"))
+            target_release = (bug_config or {}).get('target_release', [])
+            z_stream = f"{major}.{minor}.z"
+            if z_stream not in target_release:
+                errors.append(
+                    f"bug.yml 'target_release' must include '{z_stream}', "
+                    f"but found: {target_release}"
+                )
+        except Exception:
```

Ruff (0.15.15)

[warning] Do not catch blind exception: `Exception`

(BLE001)